content/en/blog/_posts/2024-09-30-gateway-api-v1.2/index.md

```diff
@@ -0,0 +1,418 @@
+---
+layout: blog
+title: "Gateway API v1.2: Protocols, Labels, and Timeouts, oh my!"
+date: 2024-05-09T09:00:00-08:00
+slug: gateway-api-v1-1
+author: Gateway API Contributors
+---
+
+![Gateway API logo](gateway-api-logo.svg)
```

Comment:
```suggestion
![gateway api](gateway-api-logo.svg)
```
(that's what the letters actually say)

Reply:
I'm going to resolve this as-is because I also think the alt-text is meant to indicate if an image is a logo...